src/components/AdminActionTile/AdminActionTile.module.scss

```diff
@@ -0,0 +1,64 @@
+.buttonDiv {
+  width: 100%;
+  float: left;
+}
+
+.container {
+  align-items: center;
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  justify-content: space-around;
+  padding: 2.5rem;
+  gap: 2.5rem;
+  border-radius: 0.625rem;
+  border: 2px solid #e5e6eb;
+  background: var(--primary-FEFCFD, #fefcfd);
+}
+
+.heading {
+  display: flex;
+  align-items: center;
+  gap: 1.25rem;
+  align-self: stretch;
+
+  h3 {
+    color: var(--primary-11356F, #11356f);
```

Comment:
Same here

Reply:
Also, a lot of these styles declared are unnecessary as they are set in globals, all unnecessary ones are:
`font-family`
`font-size`
`font-style`
and probs don't need 
`leading-trim`
`text-edge`
`line-height`
`letter-spacing`